content/docs/start/data-management/metrics-parameters-plots.md

```diff
@@ -99,29 +104,43 @@ files to be versioned by Git.
 }
 ```
 
-You can view tracked metrics with `dvc metrics show `:
+As you can see, it's a simple json file and the structure is up to you and your
+project (see [supported formats] to learn more).
+
+DVC doesn't care how you generate [metric files][metrics file] - you can create
+them whichever way you like. As long as metrics are numerical DVC can pick them
+up for visualization via the command line (as we're about to see), VS Code's
+[DVC extension] or [Studio][studio metrics].
+
+Now that DVC is aware of the [metrics file], we can visualize the metrics in the
+terminal:
 
 ```dvc
 $ dvc metrics show
 Path                    avg_prec.test    avg_prec.train    roc_auc.test    roc_auc.train
 eval/live/metrics.json  0.94496          0.97723           0.96191         0.98737
 ```
 
-[`evaluate.py`]:
-  https://github.com/iterative/example-get-started/blob/master/src/evaluate.py
-[roc-auc]:
-  https://scikit-learn.org/stable/modules/model_evaluation.html#receiver-operating-characteristic-roc
-[average precision]:
-  https://scikit-learn.org/stable/modules/model_evaluation.html#precision-recall-and-f-measures
-[metrics file]: /doc/command-reference/metrics#supported-file-formats
+<admon icon="book">
```

Comment:
Q: was it here before? do we need it? (we are trying to keep it simpler usually)

Reply:
No, it's new
Motivation: metrics don't seem to have a lot to offer in the guide now. if you read this section in the review app (we moved the comparison details away, as they are too heavy / exp oriented), it now looks like DVC offers "only" very thin functionality around metrics with `dvc metrics show`. to balance this out a bit I wanted to make it clear that this is more powerful than the simple command demonstrated, without going into cases and examples

I hope this helps shed some light on this decision!

Reply:
yep, it'a sign and one more reason to keep compare (since that's what is needed, not show, and even mentioning that show can do is not needed since it's not a primary command).

Reply:
I'm afraid you lost me here a bit🙁 
It's important the reader understands what DVC can do around metrics / params / etc. why are we presenting them to the reader.
Please see review app - compare is still here (mentioned, but condensed) - it's a reference / advanced mention. informing the user but not going through commands and outputs and taking half the page. Moving this content away is a basic premise of this PR (see [description](https://github.com/iterative/dvc.org/pull/4519#issue-1698046930)).
Please browse the review app and see how this section feels now and how the comparisons feel in their new home, around experiments.

If you still want to block or avoid this change - please let me know, and I'll just close this PR. this is "high-level" feedback / discussion, and not related to this specific admon.

Reply:
> It's important the reader understands what DVC can do around metrics / params / etc. why are we presenting them to the reader.

yes, and it feels you are improving this already in this PR in the very beginning, above, etc.

> If you still want to block or avoid this change - please let me know, and I'll just close this PR. this is "high-level" feedback / discussion, and not related to this specific admon.

tbh, not so much worried even about this PR. More like sharing some fundamentals that we've been optimizing for (some other discussions). Each PR doesn't change much. Being aligned on a framework, target audience, style, etc, etc - helps to do bigger changes efficiently.